vars/kola.groovy

```diff
@@ -41,7 +41,7 @@ def call(params = [:]) {
     // conditionally only add the `run_upgrades` stage if not explicitly
     // skipped.
     def kolaRuns = [:]
-    kolaRuns["run"] = {
+    kolaRuns["${arch}:Kola"] = {
```

Comment:
Bikeshed: for consistency, should we use `kola` (lowercase) here and the other stage names?

Reply:
can do

Reply:
done